pkg/apis/logging/v1/clusterlogging_types.go

```diff
@@ -50,8 +50,25 @@ type ProxySpec struct {
 
 // This is the struct that will contain information pertinent to Log storage (Elasticsearch)
 type LogStoreSpec struct {
-	Type              LogStoreType `json:"type"`
-	ElasticsearchSpec `json:"elasticsearch,omitempty"`
+	Type                LogStoreType `json:"type"`
+	ElasticsearchSpec   `json:"elasticsearch,omitempty"`
+	RetentionPolicySpec `json:"retentionPolicy,omitempty"`
+}
+
+//TimeUnit is a time unit like h,m,d
+type TimeUnit string
```

Comment:
No need to define since we are vendoring the EO  this should come from https://github.com/openshift/elasticsearch-operator/blob/feature-es6x/pkg/apis/logging/v1/index_management_types.go#L16